src/main/resources/templates/java/gradle_gradle/exercise/build.gradle

```
@@ -10,16 +11,14 @@ dependencies {
     implementation 'org.apache.commons:commons-lang3:3.12.0'
 }
 
-def assignmentSrcDir = "src"
-
 tasks.withType(JavaCompile) {
     options.encoding = 'UTF-8'
 }
 
 sourceSets {
     main {
         java {
-            srcDir assignmentSrcDir
+            srcDir "src"
```

Since you are using single quotation marks everywhere else I think it would make sense to also use them here.
```suggestion
            srcDir 'src'
```
Would also apply to other files.